src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/VariantAnnotationArgumentCollection.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.hellbender.cmdline.argumentcollections;
+
+import com.google.common.collect.Lists;
+import org.broadinstitute.barclay.argparser.Advanced;
+import org.broadinstitute.barclay.argparser.Argument;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class VariantAnnotationArgumentCollection implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    public VariantAnnotationArgumentCollection(List<String> defaultGroups, List<String> defaultAnnotations, List<String> defaultExclude) {
```

Comment:
`defaultExclude` -> `defaultAnnotationsToExclude` (otherwise could be misinterpreted)